templates/patterns/list_item/pattern-list-item--variant-navigation.html.twig

```diff
@@ -2,12 +2,30 @@
 /**
  * @file
  * Default implementation for the navigation list item component.
+ *
+ * @see @ecl/navigation-lists
  */
 #}
-{% include '@ecl/ec-component-list-item' with {
-  'variant': 'navigation',
-  'title': title,
-  'href': url,
-  'detail': detail,
-  'screen_reader': 'List item'|t
+
+{% set _infos = {} %}
+{% for infos_item in infos %}
+  {% set _info = {
+    label : infos_item.text,
+    icon : {
+      type : 'general',
+      name : infos_item.icon,
+      path : ecl_icon_path
+    }
+  } %}
+  {% set _infos = _infos|merge([_info]) %}
+{% endfor %}
+
+{% include '@ecl/card' with {
```

Comment:
The ticket mentions using the Navigation list composition, it would be nice if you could put a comment in the ticket or here explaining why you chose to use the card template instead.

Reply:
You are alright.

Reply:
Commented on the ticket.